packages/mdc-tabs/tab/_mixins.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@mixin mdc-icon-tab-content {
```

Comment:
Since these mixins live in the global scope, it's not clear that `mdc-icon-tab-content` and `mdc-anchor-reset` pertain to `mdc-tabs`. Usually we prefix mixins with `mdc-<componentName>` to define them. 

Furthermore, if these mixins are not intended to be reused outside of the `mdc-tabs.scss` file, then I'm not sure they need to be broken out into a separate file. Note that this applies to all `_variables` files as well as all `_mixins` files across all 3 components